### DISCUSSION TITLE
Cant send email via smtp, using sendblaster

### DISCUSSION THREAD

**Discussion by ONLOCKED1101** [postalserver/postal#1943] [Help with using Postal]
![image](https://user-images.githubusercontent.com/103117630/161969704-ef66b18e-00ee-4374-aca3-2b0f44dddfa6.png)

I keep getting this after trying to test smtp
domain is correctly configured and every TXT and CNAME record has been added.

ERROR: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 46.21.153.246:25

**Comment by johnceci-m3** [comment]
Have you checked for any necessary changes to your firewall to allow port 25 through on that IP address?  Would seem to be a firewall issue...

**Reply by ONLOCKED1101** [reply]
Do i NEED to install SSL certificate on my server, I'm using ubuntu 20

**Reply by uhlhosting** [reply]
For SMTP in Postal to be over SSL/TLS you will need SSL in postal.yml setup under smtp. RSA only supported.

**Reply by ONLOCKED1101** [reply]
Please I'm having issues configuring my SSL.,,, port 25 is not open


this is what I'm receiving 

Connecting to 46.21.153.246
4/6/2022 11:36:53 PM Connection attempt #1 - Unable to connect after 15 seconds. [15.01 sec]

LookupServer 15015ms

**Reply by ONLOCKED1101** [reply]
Can anyone help with this?

**Reply by uhlhosting** [reply]
You need to allow port 25 on firewall, what is the status of postal status and what OS you run?

**Reply by ONLOCKED1101** [reply]
I'm using ubuntu, currently firewall is denied, if I allow firewall, i wont be able to access postal web interface.

**Reply by ONLOCKED1101** [reply]
46.21.153.246 thats my host

**Reply by willpower232** [reply]
it may be that your server provider has a separate firewall which needs updating to allow port 25
